### PR TITLE
Add ovn vxlan workflow and configure WMCO jobs

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
-  Add AWS Hybrid OVN with vxlan workflow
-  Add WMCO e2e test ref that uses the AWS Hybrid OVN with vxlan workflow
- Use the WMCO e2e test ref in the WCMO configs

Ran `make validate-step-registry` and `make jobs`.

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
/approve cancel

@dcbw, @aravindhp, @JacobTanenbaum

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -39,6 +39,12 @@ tests:
     test:
     - ref: windows-e2e-operator-test
     workflow: ipi-aws-ovn-hybrid
+- as: e2e-operator-with-vxlan
```
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:42]: For correctness, I would name this `e2e-operator-custom-vxlan-port`

File: ci-operator/step-registry/ipi/aws/ovn/hybrid-with-vxlan/ipi-aws-ovn-hybrid-with-vxlan-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: ipi-aws-ovn-hybrid-with-vxlan
```
Comment (aravindhp) [inline ci-operator/step-registry/ipi/aws/ovn/hybrid-with-vxlan/ipi-aws-ovn-hybrid-with-vxlan-workflow.yaml:2]: Here too I would name it, `ipi-aws-ovn-hybrid-custom-vxlan-port` and rename everything else associated like the sub-directories in a similar fashion.

File: ci-operator/step-registry/ipi/aws/ovn/hybrid-with-vxlan/ipi-aws-ovn-hybrid-with-vxlan-workflow.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest-with-vxlan
+    - chain: ipi-install
+    post:
+    - chain: ipi-deprovision
+  documentation: |-
+    OpenShift cluster configured with Hybrid OVN network
```
Comment (aravindhp) [inline ci-operator/step-registry/ipi/aws/ovn/hybrid-with-vxlan/ipi-aws-ovn-hybrid-with-vxlan-workflow.yaml:12]: `...with vxlan port set to 4800`

File: ci-operator/step-registry/ovn/conf/hybrid-manifest-with-vxlan/ovn-conf-hybrid-manifest-with-vxlan-ref.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ovn-conf-hybrid-manifest-with-vxlan
+  from: base
+  commands: ovn-conf-hybrid-manifest-with-vxlan-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The ovn hybrid manifest step adds the configuration for the hybrid networking with vxlan to set of manifests for installation
```
Comment (aravindhp) [inline ci-operator/step-registry/ovn/conf/hybrid-manifest-with-vxlan/ovn-conf-hybrid-manifest-with-vxlan-ref.yaml:10]: `......with vxlan port set to 4800...` needs to be mentioned somewhere.

**Comment by JacobTanenbaum** [comment]
it looks like you need to run make `jobs again`

**Comment by JacobTanenbaum** [comment]
@ravisantoshgudimetla would it be better to make a step that takes a hybrid manifest and adds the vxlan configuration too it, as opposed to generating the whole hybrid manifest file? appending that smaller step to the larger job of deploying the hybrid network?

**Comment by ravisantoshgudimetla** [comment]
> @ravisantoshgudimetla would it be better to make a step that takes a hybrid manifest and adds the vxlan configuration too it, as opposed to generating the whole hybrid manifest file? appending that smaller step to the larger job of deploying the hybrid network?

@JacobTanenbaum - I thought about it, however my bash-foo isn't great. The last time I tried, I got an error like this, not sure if I was missing something:

```
sed  'hybridOverlayConfig\: /a hybridOverlayVXLANPort:4800' /tmp/sample.txt
sed: -e expression #1, char 2: extra characters after command
```

with sample.txt having contents of network.yaml. So, I thought of going with creating an independent step with a new hybrid overlay manifest generated. I can change it if you prefer this way.

**Comment by aravindhp** [comment]
/approve

for the WMCO bits

**Review by petr-muller — APPROVED** [review]

**Pull request merged**